src/coreclr/inc/cordebuginfo.h

```diff
@@ -380,10 +380,11 @@ class ICorDebugInfo
         VARARGS_HND_ILNUM   = -1, // Value for the CORINFO_VARARGS_HANDLE varNumber
         RETBUF_ILNUM        = -2, // Pointer to the return-buffer
         TYPECTXT_ILNUM      = -3, // ParamTypeArg for CORINFO_GENERICS_CTXT_FROM_PARAMTYPEARG
+        IMPLICIT_THIS       = -4, // Implicit (non-user) this arg
 
-        UNKNOWN_ILNUM       = -4, // Unknown variable
+        UNKNOWN_ILNUM       = -5, // Unknown variable
 
-        MAX_ILNUM           = -4  // Sentinel value. This should be set to the largest magnitude value in th enum
+        MAX_ILNUM           = -5  // Sentinel value. This should be set to the largest magnitude value in th enum
```

Comment:
This enum is part of the JIT/EE interface and runtime debugger contracts. Changing it has impact in number of places. For example, this is R2R format breaking change among other things.

Reply:
Ah, didn't realize that, I can avoid changing it, although, it's a bit weird that we don't have "implicit this" in it as it's the same "hidden" arg as e.g. return-buffer or generic context (except cases where "this" is explicit user arg)

Reply:
"implicit this" is an explicit arg at IL level. It is not the same as return-buffer or generic context that do not exist at IL level.